order-status/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This document explains the structure of a STAT **Order Status** request. Operation returns a list/array of statuses that have affected an order.  The most recent status code is the current status of the order.
+
+
+Endpoint: GET /order/{id}/status
+
+## Order Status Request
+
+List operation only in specced API.
+
+## Order Status Response
+
+Array of:
+
+| Field Name | Type | Description |
+| ---------- | ---- | ----------- |
+| timestamp  | datetime | Timestamp for the order status (required) |
+| status_code | string | Enumerated status code (required) |
+| reason_code | string | Enumerated reason code for why the status was set (optional) |
+| reason_text | string | Textual description for why the status was set (optional) |
```

Comment:
Room questions the usability & usefulness of reason_text.

Reply:
Depending on use-case may need an i18n string in a UI and not use whatever the provider returned